src/libkomwm.py

```diff
@@ -72,18 +72,30 @@ def query_style(args):
 
     results = []
     for zoom in range(minzoom, maxzoom + 1):
-        runtime_conditions_arr = []
-
+        all_runtime_conditions_arr = []
         # Get runtime conditions which are used for class 'cl' on zoom 'zoom'
         if "area" not in cltags:
-            runtime_conditions_arr.extend(style.get_runtime_rules(clname, "line", cltags, zoom))
-        runtime_conditions_arr.extend(style.get_runtime_rules(clname, "area", cltags, zoom))
+            all_runtime_conditions_arr.extend(style.get_runtime_rules(clname, "line", cltags, zoom))
+        all_runtime_conditions_arr.extend(style.get_runtime_rules(clname, "area", cltags, zoom))
         if "area" not in cltags:
-            runtime_conditions_arr.extend(style.get_runtime_rules(clname, "node", cltags, zoom))
+            all_runtime_conditions_arr.extend(style.get_runtime_rules(clname, "node", cltags, zoom))
 
-        # If there is no any runtime conditions, do not filter style by runtime conditions
-        if len(runtime_conditions_arr) == 0:
-            runtime_conditions_arr.append(None)
+        if len(all_runtime_conditions_arr) == 0:
+            # If there is no runtime conditions, do not filter style by runtime conditions
+            runtime_conditions_arr = [None]
```

Comment:
Declare runtime_conditions_arr = [] before if blocks?

Reply:
Why? I see it'll be a bit more kind of robust.. But it will be either re-assigned or appended anyway, so it'll be just an extra assignment that is always done, but is not really needed, so I decided not to do it...

Reply:
Because it is used outside of the ifs scope. Probably, I'm _fixed_ on static-typing languages, but I was confused what "runtime_conditions_arr" var is used below. May its some var from global scope ...

Reply:
Ok I've changed it as per your suggestion! PTAL